MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+recursive-include openfisca_web_api/i18n *.mo
+include *.md
+include LICENSE
+include *.ini
```

Comment:
I'm the author of the above blog post with the fix and in the past few months discovered what I think is a better solution -- essentially make sure you are using setuptools, not distutils, and that you're using the most updated version. I've updated the blog post with the details. Hope that's helpful!